### ISSUE TITLE
DVC binaries are not signed

### ISSUE THREAD

**Issue by ramank775** [iterative/dvc#4180]
I am exploring the DVC as the data versioning tool in my organization but found that binaries are not signed. Which is a blocker as per the organization rules. 
Is there any plan to signed the binaries?

**Comment by shcheklein** [comment]
@ramank775 hi! what platform are using in your org? Also, have you considered using `pip`, `snap`, `brew`, `conda` options as a workaround?

**Comment by efiop** [comment]
For the record: for deb/rpm/exe packages we need https://github.com/iterative/dvc/issues/936

**Comment by ramank775** [comment]
We are mainly using Windows/Linux as development machine.

**Comment by efiop** [comment]
@ramank775 Have you considered using pip or conda packages?

**Comment by ramank775** [comment]
` So are you using exe and deb/rpm packages right now?`
- yes

We have a review process for any tools before we can actually use it. The team has blocked the tool by reviewing the official website and the recommended method is exe and binaries. I will try to reach out to them for alternative methods but not very sure if they approve them or not.

**Comment by skshetry** [comment]
Duplicate of #936
